ui/src/main/webapp/src/app/shared/validators/project-name-not-exists.validator.ts

```diff
@@ -21,6 +24,10 @@ export class ProjectNameNotExistsValidator implements Validator {
         let projectService = this.projectService;
 
         return new Promise(resolve => {
+            if (this.editMode) {
+                resolve(null);
```

Comment:
I don't think this is correct. If you edit project name, you want to do validation too. There is just one exception - if you type the same project name as is currently used, it should pass.